src/Admin/AdminExtensionInterface.php

```diff
@@ -129,11 +113,9 @@ public function configureActionButtons(
     public function configureDefaultFilterValues(AdminInterface $admin, array &$filterValues): void;
 
     /*
-     * NEXT_MAJOR: Uncomment this method
-     *
      * Returns a list of default sort values
      */
-    // public function configureDefaultSortValues(AdminInterface $admin, array &$sortValues): void;
+    public function configureDefaultSortValues(AdminInterface $admin, array &$sortValues): void;
 }
 
 class_exists(\Sonata\Form\Validator\ErrorElement::class);
```

Comment:
```suggestion
```

I think we don't need it anymore here. It was use to call ErrorElement from CoreBundle to trigger aliases. Should be delete in other files too.

Reply:
Fixed